src/language-js/embed.js

```diff
@@ -508,12 +510,9 @@ function hasLanguageComment(node, languageName) {
   // we will not trim the comment value and we will expect exactly one space on
   // either side of the GraphQL string
   // Also see ./clean.js
-  return (
-    node.leadingComments &&
-    node.leadingComments.some(
-      comment =>
-        comment.type === "CommentBlock" && comment.value === ` ${languageName} `
-    )
+  return hasLeadingComment(
+    node,
+    comment => isBlockComment(comment) && comment.value === ` ${languageName} `
```

Comment:
`comment.value.trim() ===languageName`?

Reply:
Unless you'd like to match the comment exactly with a single space

Reply:
https://github.com/prettier/prettier/pull/4395#commitcomment-28780334 😉

https://github.com/prettier/prettier/blob/3caa667240616cc582291a40d3fa3ae0131bb4f3/src/language-js/embed.js#L508-L511

Reply:
Well look at me being all duplicative :)